ParabolInc/parabol
# feat(sprint-poker): Search GitLab issues

Fix #6045 

### To test

- Integrate with GitLab. If you can't see the GitLab integration provider make sure you've added the `gitlab` feature flag. If that doesn't work, run `yarn postdeploy` to make sure the `cloudProvider` is added
- Make sure you're part of the GitLab team Parabol1
- Create a Sprint Poker meeting, go to the Scope phase, and see your GitLab issues
- Search for an issue in the search input and see that the issues filter

Note: with GitLab's search API, no results are returned if the search term doesn't match the issue title. So, if an issue is called Test and you type in "Tes", the issue may not show up.
